R/project-element.R

```diff
@@ -0,0 +1,124 @@
+#' @title Add project element 
+#' @description This function creates a project node within the parent element that 
+#' contains all the required elements for the project section of an EML document.
+#' This function can be used in combination with add_personnel
+#' and add_funding. Add_personnel can be used to generate the project_personnel 
```

Comment:
don't capitalize a function name and use code inline formatting wherever you make a reference to code in doc prose

`\code{add_personnel}`